lab5.py

```diff
@@ -0,0 +1,103 @@
+from enum import Enum
+
+class City(Enum):
+    LVIV = 1
+    KYIV = 2 
+    BURSHTYN = 3
+
+class WeatherType(Enum):
+    SUNNY = 0
+    CLOUDY = 1
+    RAINY = 2
+    FOGGY = 3
+
+class Weather(object):
+
+    def __init__(self, day, city, country, temp, humidity, wind_speed, weather_type):
+        self.day = day
+        self.city = city
+        self.country = country
+        self.temp = temp
+        self.humidity = humidity
+        self.wind_speed = wind_speed
+        self.weather_type = weather_type
+        
+    def __del__(self):
+        print("Cleanup has been completed")
+        
+    def getAll(self):

```

Comment:
it's going to be clearer what we're getting

Reply:
If we write "get_all_parameters" instead of "getAll" , I think it will be clearer what exactly we should get.

Reply:
It has been already suggested and I committed that change. Anyway, thanks